osl/interface_linux.go

```diff
@@ -62,6 +63,13 @@ func (i *nwIface) Master() string {
 	return i.master
 }
 
+func (i *nwIface) MacAddress() net.HardwareAddr {
```

Comment:
My understanding is that so far only the `osl` functions which get invoked from `libnetwork` are ported to the respective platform package, as this is needed to allow building libnetwork on the other platforms.

This function is not being called outside of the `osl` package.